tests/test_client_response.py

```diff
@@ -1243,3 +1243,22 @@ def test_response_links_empty(loop: Any, session: Any) -> None:
     )
     response._headers = CIMultiDict()
     assert response.links == {}
+
+
+def test_response_not_closed_after_get_ok() -> None:
```

Comment:
You could use `mocker.spy()` on `response.raise_for_status` method to verify that it wasn't called.

https://github.com/pytest-dev/pytest-mock#spy